mmv1/products/vertexai/terraform.yaml

```diff
@@ -69,9 +69,6 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           address_name: "address-name"
           kms_key_name: "kms-name"
           network_name: "network-name"
-        test_vars_overrides:
```

Comment:
note for myself: we use `skip_test` so this is just noise

Reply:
Yep, sorry, should have made a note on this one